doc/Create-resources-using-aso.md

```diff
@@ -0,0 +1,200 @@
+# Preparing Infrastructure for ARO HCP Using ASO
+
+We need to initialize the prerequisite infrastructure to create an ARO-HCP cluster:
+
+* This can be done using the `az` CLI command, as shown here: [Creating an HCP via Cluster Service](https://github.com/Azure/ARO-HCP/blob/main/cluster-service/cluster-creation.md)  
+* In this document, we demonstrate how to achieve the same using `k8s` and [Azure Service Operator v2](https://azure.github.io/azure-service-operator/) with a declarative approach.
+
+## Prerequisites
+
+We expect the following:
+
+* A `kind` cluster is being used (or any other Kubernetes cluster)  
+* The following tools are installed:  
+  * `az` CLI (or a `sp.json` file is already created)  
+  * `kubectl` – these are prerequisites for creating the ARO-HCP infrastructure using the declarative approach  
+  * `helm` – also required for setting up the infrastructure using the declarative approach
+
+1. Create azure account service principal and store it in json file sp.json as follow
+```bash
+AZURE_SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+if [ ! -f sp.json ] ; then
+    let "randomIdentifier=$RANDOM*$RANDOM"
+    servicePrincipalName="msdocs-sp-$randomIdentifier"
+    roleName="Contributor"
+    echo "Creating SP for RBAC with name $servicePrincipalName, with role $roleName and in scopes /subscriptions/$AZURE_SUBSCRIPTION_ID"
+    az ad sp create-for-rbac --name $servicePrincipalName --role $roleName --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID > sp.json
+fi
+```
+2. Install cert-manager using a Helm chart
+This is required for non-OCP clusters, as OCP clusters include a different built-in cert-manager.
+```bash
+helm repo add jetstack https://charts.jetstack.io --force-update
+helm repo update
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set crds.enabled=true
+````
+3. Install ASO2 using a Helm chart 
```

Comment:
Install CAPZ using helm chart and add URL https://cluster-api.sigs.k8s.io/user/quick-start.html ref.  ASO come with CAPZ by default.